vision.md

```diff
@@ -1,52 +1,41 @@
-Stated simply...
+## Stated simply...
 
-**NUnit 3 is a world-class platform for individuals and teams using a test-based approach to .NET software development.**
+**NUnit aims to be a best in breed test framework that enables end to end testing of .NET projects by individuals and teams. It is unopinionated and supports a number of different kinds of tests. NUnit allows for alternative approaches to testing and is usable by team members in a variety of roles.**
 
```

Comment:
Ah ok, this is covered below. I'd personally merge the last two sentences. Perhaps this was clear as it was to everyone else though? Maybe it's just my non-academic perspective on testing that couldn't parse this?

> NUnit aims to be a best in breed test framework that enables end to end testing of .NET projects by individuals and teams. It is unopinionated, supports a number of different approaches to testing, and is usable by team members in a variety of roles.